src/app/modules/main/profile_section/language/locale_table.nim

```diff
@@ -1,44 +1,50 @@
 import tables
 
 type
+  State* {.pure.} = enum
+    Alpha  # 30-90% translations done
+    Beta   # >90% translations done
+    Stable # >90% translations done and reviewed
+
   Description* = object
     name*: string
     native*: string
     flag*: string
+    state*: State
 
 let localeDescriptionTable* = {
-    "ar": Description(name: "Arabic", native: "العربية", flag: ""),
```

Comment:
Do we always have to set the state manually here? Thinking if it could be improved somehow and evaluated from the .qm file... but I don't see any easy way :/

Reply:
Yeah, I don't like it either, but didn't find any easy way to automate it. That's something we could come up with later if it becomes really annoying.

I was thinking about the script that will generate metadata near to *.qm files (let's say de.qm -> de.qm.meta) that will hold the state. That would be easy for alpha vs beta but not for beta vs stable, cause *.ts files do not contain information if the given translation was reviewed, it's only Lokalise metrics I believe.

Reply:
Yeah good idea; not sure but doesn't Lokalise provide some sort of API we could tap into?

Reply:
Don't know, probably yes, but haven't investigated it. It would enable us to write a script that downloads translations and generates proper metadata for it. This is something worth considering :+1: